test/corpus/comments_and_notes.txt

```diff
@@ -77,3 +77,30 @@ blank lines
     (comment))
   (journal_item
     (comment)))
+
+================================================================================
+metadata tags
+================================================================================
+
+2012-03-10 * KFC
+    Expenses:Food                $20.00
+    Assets:Cash
+      ; :TAG1:TAG2:TAG3:
+
+--------------------------------------------------------------------------------
+
+(source_file
+  (journal_item
+    (xact
+      (plain_xact
+        (date)
+        (status)
+        (payee)
+        (posting
+          (account)
+          (amount
+            (commodity)
+            (quantity)))
+        (posting
+          (account))
+        (note)))))
```

Comment:
This should be `(note (tag) (tag) (tag))` but not sure why tags aren't detected.

Reply:
It looks like this grammar takes manual control of whitespace by setting `extras: ($) => []`. Extras defaults to whitespace and controls what tokens may appear anywhere in the grammar without needing to write them explicitly. With it disabled, you may need to add `optional($.whitespace)` in places where whitespace is valid